crates/rspack_binding_values/src/value_ref.rs

```diff
@@ -0,0 +1,71 @@
+use std::ptr;
+
+use napi::sys::{napi_env, napi_value};
+use napi::{bindgen_prelude::ToNapiValue, check_status, sys, Result};
+
+pub struct Ref {
```

Comment:
@h-a-n-a need you checkout this

Reply:
Please add some comments to this file, including the place it originally copied from and the changes applied in this file.